src/main/clojure/clara/macros.clj

```diff
@@ -43,33 +43,25 @@
    (coll? source) (seq source)
    :else (throw (IllegalArgumentException. "Unknown source value type passed to defsession"))))
 
```

Comment:
Looking back at this I'm not sure why we don't add a docstring to the var def the same way as in Clojure.  That's the status quo though, we don't need to make changes in this PR.